webapp/src/components/schedule/WeekCalendar.js

```diff
@@ -87,16 +122,23 @@ class WeekCalendar extends PureComponent {
         };
 
         let icsDownload = (
-            <div className="ics-button">
-                <Button label="Download Calendar" className="ui-button-info"
-                        onClick={this.downloadICS.bind(this, this.state.subsections)}
-                        disabled={this.props.empty}/>
-            </div>
+            <Button label="Download Calendar" className="ui-button-info"
+                    onClick={this.downloadICS.bind(this, this.state.subsections)}
+                    disabled={this.props.empty}/>
+        );
+
+        let toGCalendar = (
+            <Button id="gcalendar-button" label="Add to Google Calendar" className="ui-button-info"
+                    onClick={this.addToGCalendar.bind(this, this.state.subsections)}
```

Comment:
if you factor out the add method completely you can just use addEvents directly here - if you need help ask me about this one, context is a tricky javascript thing